templates/terraform/schema_property.erb

```diff
@@ -143,13 +143,13 @@
 <% end -%>
 <% unless property.conflicting().empty? -%>
 <% conflicting_props = property.conflicting().map(&:name).map(&:underscore) -%>
-    ConflictsWith: <%= go_literal(conflicting_props) -%>,
+    ConflictsWith: <%= go_literal(conflicting_props.reject {|sp| property_for_schema_path(sp, object).nil? }) -%>,
```

Comment:
It sounds like we can do `.map{|sp| property_for_schema_path(sp, object)}.reject{&:nil?}` (or whatever the syntax for checking nil is), right? That feels a little cleaner.

Reply:
mmm we want the original schema path though, not the mapped property - i can create an exclude_schema_path? helper to make it cleaner though

Reply:
Oh got it- ignore me, I thought that was what it returned.